### PR TITLE
DEVX-4574: Document push structure

### PR DESCRIPTION
**Pull request by abdulajet**
## Description

An example of the push payload for reference. Added to the in-app voice guides
https://jira.vonage.com/browse/DEVX-4574

Page: https://nexmo-developer-pr-3504.herokuapp.com/client-sdk/in-app-voice/guides/push-notification-payload

### CODE REVIEW

File: _documentation/en/client-sdk/in-app-voice/guides/push-notification-payload.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+# Push Notification Payload Example
+
+This is an example of the push payload that is send to the Client SDK from Vonage. The key part to your application being in the `push_info` object. 
+
+```json
+{
+  "application_id": "e2dd57b5-5071-4506-8827-8efa5b96fc9d", // Your Vonage Application ID
+  "timestamp": "2021-01-06T17:18:19.093Z",
+  "user_id": "USR-6c38a318-a24d-43c4-a667-563696416040",
+    "push_info": { // Information for the push notification
```
Comment (pardel) [inline _documentation/en/client-sdk/in-app-voice/guides/push-notification-payload.md:16]: I think this needs to be aligned with "user_id"

File: _documentation/en/client-sdk/in-app-voice/guides/push-notification-payload.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    "conversation": {
+      "conversation_id": "CON-b24b963a-c50c-46b9-ac26-871d408f2f7c", // Your conversation name
+      "name": "NAM-23c60ce9-77b4-4c99-aaec-cd1ad6cc0e31" // Your conversation ID
+    },
+    "initiator": {
+      "invited": {
+        "isSystem": 1
+      }
+    },
+    "invited_by": "<null>",
```
Comment (pardel) [inline _documentation/en/client-sdk/in-app-voice/guides/push-notification-payload.md:61]: We should make the engineering aware of the <null> coming through here - I'm guessing it should be "system".

**Review by pardel — CHANGES_REQUESTED** [review]
Just one tiny change.

File: _documentation/en/client-sdk/in-app-voice/guides/push-notification-payload.md (hunk 2 above)
Comment (abdulajet) [reply _documentation/en/client-sdk/in-app-voice/guides/push-notification-payload.md:61]: Ill let ashley know

**Review by pardel — CHANGES_REQUESTED** [review]
a small change

**Review by pardel — APPROVED** [review]

**Pull request merged**